src/dmd/semantic3.d

```diff
@@ -279,19 +279,9 @@ private extern(C++) final class Semantic3Visitor : Visitor
         }
 
         uint oldErrors = global.errors;
+        auto fds = FuncDeclSem3(funcdecl,sc);
```

Comment:
Can this be made `const` or are we expecting it to change?

Reply:
It will eventually.

Reply:
ok.